docker/nginx/default.conf

```diff
@@ -22,18 +22,36 @@ server {
     send_timeout                300;
 
     location /static/ {
+        if ($http_x_forwarded_proto = "http") {
+            return 301 https://$host$request_uri;
+        }
+        if ($http_x_forwarded_proto = "https") {
+            add_header Strict-Transport-Security "max-age=3600;" always;
```

Comment:
FWIW - The header was showing up even on HTTP requests - so I think it will anyway (which was surprising) 

Regardless I think it's ok even if we do serve the STS header on HTTP requests.